include/zephyr/drivers/clock_control/stm32_clock_control.h

```diff
@@ -281,15 +281,15 @@ struct stm32_pclken {
 
 /** Device tree clocks helpers  */
 
-#define STM32_CLOCK_INFO(clk_index, id)					\
+#define STM32_CLOCK_INFO(clk_index, node_id)				\
```

Comment:
you can also simplify `STM32_INST_CLOCK_INFO` by using this one now

Reply:
Right! That a good sign about macro usability :-), and a great way to test.
Thanks, done.